elements/xml_form_elements.module

```diff
@@ -300,6 +300,17 @@ function xml_form_elements_form_element_tabs_ajax_add(FormElement $element, arra
   $new_tab = clone $tab;
   $new_tab->eachDecendant(function($element) {
     $element->default_value = $element->getOriginalDefaultValue();
+                            if ($element->controls['#type'] == 'tags') {
```

Comment:
@knjackson30974

The inconsistent whitespace could be cleaned up.

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.](http://www.discoverygarden.ca) | Managing Digital Content*